CMakeSettings.json

```diff
@@ -1,210 +1,144 @@
 ﻿{
-    "configurations": [
+  "configurations": [
+    {
+      "name": "x64-MSVC-Analysis",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "buildRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\build\\${name}",
+      "installRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\install\\${name}",
+      "cmakeCommandArgs": "",
```

Comment:
"cmakeCommandArgs": "-DCMAKE_VERBOSE_MAKEFILE=ON -DVCPKG_TARGET_TRIPLET=x64-windows-static -DBUILD_SHARED_LIBS=OFF"

As vcpkg when installing ttauri, calls cmake with both the tripplet and build_shared_libs. It seems okay to have to set both in the cmakesettings.json as well.

Reply:
Ok, switching to static builds by default.